packages/trader/src/AppV2/Utils/positions-utils.ts

```diff
@@ -20,6 +20,24 @@ export const filterPositions = (positions: (TPortfolioPosition | TClosedPosition
         return splittedFilter.includes('main_title' in config ? config.main_title : config.name);
     });
 };
+const encryptionConfig = {
```

Comment:
I don't know how to name it better, please help) Here it is the correspondence between the name of the contract that the user sees and its encoding in the backend

Reply:
`contractTypesConfig` :)